lib/models/Event.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose');
+
+const schema = new mongoose.Schema({
+  recipeId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Recipe',
+    required: true
+  },
+  dateOfEvent: {
+    type: Date,
+    required: true
+  },
+  notes: String,
+  rating: {
+    type: Number,
+    required: true,
+    min: 0,
+    max: 5
+  }
+});
+
+schema.virtual('day')
+  .get(() => {
+    return this.dateOfEvent.getDate();
+  });
+
```

Comment:
The only way i could see making this more concise is by doing what you did here on line 24 to both month and year, to make it just one line of code!